metrics/src/main/java/se/fortnox/reactivewizard/metrics/Metrics.java

```diff
@@ -36,10 +37,19 @@ public static Metrics get(String name) {
         return measure(observable, NOOP);
     }
 
+    public <T> Flux<T> measure(Flux<T> observable) {
```

Comment:
I saw that you had to port some different existing classes to handle Flux instead of Observable. Perhaps it would be wise to use the parent interface https://www.reactive-streams.org/reactive-streams-1.0.3-javadoc/org/reactivestreams/Publisher.html for some of these, so that we can use adapters and have user code using different implementations? There are some different flavors.

Reply:
reactor-netty uses its own metrics system so eventually we could use that entirely and wouldn't even need to add support for Flux here but It seems like a good change to have so I will switch anyway